docs/conf.py

```diff
@@ -22,9 +22,17 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+"""Sphinx configuration."""
+
 from __future__ import print_function
 
 import os
+import sys
+
+import sphinx.environment
+
+# Plug example application into module path
+sys.path.append('examples')
```

Comment:
I'm not sure it makes sense to add an example app to Invenio-Mail. If we add one, is it then just a view which will send an email when you press a button?